packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -13,7 +13,7 @@ import '_functions_io.dart' if (dart.library.html) '_functions_web.dart';
 import 'style_sheet.dart';
 import 'widget.dart';
 
-const List<String> _kBlockTags = <String>[
```

Comment:
Why drop the `const`?

Reply:
Because of adding additional tags the list cannot be const anymore.